### PR TITLE
New sprites for requests, pod, and gulag wall consoles

### PR DESCRIPTION
**Pull request by DrDiasyl**
## About The Pull Request

Changes ancient sprites for requests _(don't confuse them with cargo requests computer)_ and pod wall consoles, to the new polished ones with the same style as the newscaster. Also makes the gulag/pod console actually use overlays, instead of having the screen melted into its icon (so now when it has no electricity it screen turns off).
Now pods and gulag console's screen changes when they get emagged.

Requests console:
req_console
req_0 big
req_1 big
req_2 big
req_3 big

Pod console:
pod_console
pod_0 big

Gulag item retrieval console:
gulag_0 big
gulag_console

Emagged gulag/pod console:
emagged big

## Why It's Good For The Game
The old sprites of them are really ancient, have bad looks and that old isometric perspective.
old gulag and pod console
old req console

Having new sprites is always great for people's eyes. This also will make the requests console more noticeable, that way _maybe_ people will start using it more often.
Making them use overlays instead of having the screen icon merged into the base icon is good because it gives more dynamic, as when electricity turns out it signals more to the players that it's not functional right now.

## Changelog

:cl: DrDiasyl aka DrTuxedo#0931
imageadd: Requestions consoles got a new sprite!
imageadd: Gulag consoles got a new sprite!
imageadd: Escape and assault pod consoles got a new sprite!
qol: Now the gulag and pods consoles use overlays, instead of having turned ON screen in their base icon.
qol: Now the gulag and pods consoles screen change when they are emagged.
qol: Requests console now shows examine prompt on how to open their panel. Also shows a prompt if they were hacked.
/:cl:

### CODE REVIEW

**Comment by LSommers** [comment]
I... actually don't hate these.
Nice!

**Comment by kinneb** [comment]
the wallening is upon us....

**Review by san7890 — CHANGES_REQUESTED** [review]
> Production Consoles (unused)

In an attempt to not further bloat the rsc (resources file) from being larger than it has to be, would you mind removing those sprites from this PR? You are more than welcome to PR them to our repository for unused sprites in case they must be retrieved one day: https://github.com/tgstation/SS13-sprites

**Comment by DrDiasyl** [comment]
> > Production Consoles (unused)
> 
> In an attempt to not further bloat the rsc (resources file) from being larger than it has to be, would you mind removing those sprites from this PR? You are more than welcome to PR them to our repository for unused sprites in case they must be retrieved one day: https://github.com/tgstation/SS13-sprites

We have one? Well then, I will move it there

**Comment by DrDiasyl** [comment]
@san7890 done, I did remove it with few others unused sprites in the terminals.dmi

File: code/game/machinery/requests_console.dm [hunk 1]
```diff
@@ -108,16 +108,22 @@ GLOBAL_LIST_EMPTY(req_console_ckey_departments)
...
-/obj/machinery/requests_console/update_icon_state()
-	if(open)
-		icon_state = "[base_icon_state]_[hackState ? "rewired" : "open"]"
-		return ..()
-	icon_state = "[base_icon_state]_off"
-	return ..()
+/obj/machinery/requests_console/examine(mob/user)
+	. = ..()
+	if(!open)
+		. += span_notice("It looks like you can pry open the panel with <b>crowbar.</b>")
```
Comment (san7890) [inline code/game/machinery/requests_console.dm:114]:
```suggestion
		. += span_notice("It looks like you can pry open the panel with <b>a crowbar</b>.")
```

File: code/game/machinery/requests_console.dm [hunk 2]
```diff
@@ -108,16 +108,22 @@ GLOBAL_LIST_EMPTY(req_console_ckey_departments)
...
+	else
+		. += span_warning("Its panel is pried open, you can close it with <b>crowbar.</b>")
```
Comment (san7890) [inline code/game/machinery/requests_console.dm:116]:
```suggestion
		. += span_warning("The panel was pried open, you can close it with <b>a crowbar</b>.")
```

File: code/game/machinery/requests_console.dm [hunk 3]
```diff
@@ -108,16 +108,22 @@ GLOBAL_LIST_EMPTY(req_console_ckey_departments)
...
+
+	if(hackState)
+		. += span_warning("The console seems to be tampered with to <b>send extreme priority messages.</b>")
```
Comment (san7890) [inline code/game/machinery/requests_console.dm:119]:
```suggestion
		. += span_warning("The console seems to have been tampered with!")
```

the last part of that sentence doesn't make any sense for examine imo

**Review by san7890 — CHANGES_REQUESTED** [review]
grammar and formatting nits

**Pull request merged**